app/src/main/java/com/example/chooseyouowncocktail_g2/DrinkList.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.chooseyouowncocktail_g2
+
+import android.graphics.drawable.Drawable
+
+data class Beer(
```

Comment:
sicocme questa data class la useremo per l'interno progetto mentre l'object no, lo sposterei in una cartella chiamata model, dove metteremo tutte le data class che ci servono per la ui/app